src/containerapp/azext_containerapp/custom.py

```diff
@@ -2012,7 +2012,7 @@ def containerapp_up(cmd,
     image = _reformat_image(source, repo, image)
     token = None if not repo else get_github_access_token(cmd, ["admin:repo_hook", "repo", "workflow"], token)
 
-    if image and "mcr.microsoft.com/azuredocs/containerapps-helloworld:latest" in image.lower():
+    if image and "mcr.microsoft.com/azuredocs/containerapps-helloworld" in image.lower():
         ingress = "external" if not ingress else ingress
```

Comment:
Good to see some defaults added but can we also make sure the Output response includes the FQDN, currently not seeing it.